### PR TITLE
Hw 23 1

### PR DESCRIPTION
**Pull request by semyonich**

### CODE REVIEW

**Review by mfedenko09 — APPROVED** [review]

File: src/main/java/core/basesyntax/dao/jdbc/CarDaoJdbcImpl.java [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement carStatement = connection.prepareStatement(getCarQuery)) {
+            carStatement.setLong(1, id);
+            ResultSet carResultSet = carStatement.executeQuery();
+            while (carResultSet.next()) {
+                car = makeCar(carResultSet);
+            }
+            List<Driver> thiCarDrivers = getCarDrivers(id);
+            car.setDrivers(thiCarDrivers);
+        } catch (SQLException e) {
```
Comment (semyonich) [inline src/main/java/core/basesyntax/dao/jdbc/CarDaoJdbcImpl.java:60]: let's fetch and set drivers only if carResultSet.next() == true. Also let's create Car link inside try block and add return statement here too(Warning from prev. HW)

Comment (semyonich) [reply src/main/java/core/basesyntax/dao/jdbc/CarDaoJdbcImpl.java:60]: done

**Review by boroda4436 — CHANGES_REQUESTED** [review]
There are some classes from previous HW presented. Please remove not required files from this PR

**Comment by semyonich** [comment]
> There are some classes from previous HW presented. Please remove not required files from this PR

do you mean Application class ? because any other needless classes I cant find here?

**Pull request closed without merging**